File: Leetcode 119. Pascal's Triangle II/lc 119.java
class Solution {
    public List<Integer> getRow(int rowIndex) {
        List<Integer> ans=new ArrayList<>();
        ans.add(1);
        long a=1;

        for(int col=1;col<rowIndex+1;col++){
            a*=rowIndex+1-col;
            a/=col;
            ans.add((int)a);
        }
        return ans;
    }
}

File: Leetcode 119. Pascal's Triangle II/lc 119.cpp
class Solution {
public:
    vector<int> getRow(int rowIndex) {
        vector<int> res;
        long long ans=1;
        res.emplace_back(1);
        rowIndex++;
        for(int col=1;col<rowIndex;col++){
            ans*=rowIndex-col;
            ans/=col;
            res.emplace_back(ans);
        }
        return res;
    }
};
